src/aap_eda/services/project/scm.py

```diff
@@ -211,6 +214,7 @@ def clone(
                 key_file.close()
             if gpg_key_file:
                 gpg_key_file.close()
+                gpg_home_dir.cleanup()
```

Comment:
@bzwei how about 
```
    if gpg_home_dir:
       gpg_home_dir.cleanup()
```

Reply:
gpg_key_file and gpg_home_dir are created in the same block. No need to add an if condition